.major-version

```diff
@@ -0,0 +1 @@
+0
```

Comment:
What is this?

Reply:
No idea. took it from one of the others. deleted